src/App.jsx

```diff
@@ -1,10 +1,17 @@
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import NavBar from './components/Navbar';
+import Myprofile from './components/Routes/Myprofile';
+import Rockets from './components/Routes/Rockets';
 
 function App() {
   return (
-    <>
+    <Router>
       <NavBar />
-    </>
+      <Routes>
+        <Route path="/Profile" element={<Myprofile />} />
+        <Route path="/" element={<Rockets />} />
+      </Routes>
+    </Router>
```

Comment:
- Nice job implementing the routes. Kindly check on the order of the routes as per the design and as an optional feature you can only add the mission route path element to avoid too many conflicts when merging but ignore the import statement for the mission, I will add that